src/Features/Onboarding.php

```diff
@@ -644,19 +643,42 @@ public static function delete_themes_transient() {
 		delete_transient( self::THEMES_TRANSIENT );
 	}
 
+	/**
+	 * Check if the current page is one of the WC Admin pages.
+	 *
+	 * @return bool
+	 */
+	public static function is_woocommerce_admin_page() {
+		$is_wc_admin_page = false;
+		$current_screen   = get_current_screen();
+		if ( null !== $current_screen ) {
+			$is_wc_admin_page = 'woocommerce_page_wc-admin' === $current_screen->id;
+		}
+
+		return $is_wc_admin_page;
+	}
```

Comment:
Could [`PageController::get_current_page()`](https://github.com/woocommerce/woocommerce-admin/blob/cfeb5a163561ff98343e4da0d56e768a454ccac5/src%2FPageController.php#L201) work here instead?

Reply:
@jeffstieler Thank you for the suggestion 👍  Yup, I think `PageController::get_current_page()` is more reliable for the purpose. Change is in https://github.com/woocommerce/woocommerce-admin/pull/5771/commits/d6e1ad18f3533ef4b70cd79d30045dbc62d4f253

Reply:
@jeffstieler Updated logic to include the setting only if you are on WooCommerce -> Home or setup wizard page per Timmy's [suggestion](https://github.com/woocommerce/woocommerce-admin/issues/5702#issuecomment-733213233)